drivers/include/periph/i2c.h

```diff
@@ -140,147 +145,110 @@ int i2c_init_master(i2c_t dev, i2c_speed_t speed);
  * @param[in] dev       I2C device to access
  *
```

Comment:
missing `@param[in] speed`. What's the purpose of adding the speed to the `i2c_acquire`?